plugins/inputs/apache/apache.go

```diff
@@ -61,21 +91,53 @@ func (n *Apache) Gather(acc telegraf.Accumulator) error {
 	return outerr
 }
 
-var tr = &http.Transport{
-	ResponseHeaderTimeout: time.Duration(3 * time.Second),
-}
+func (n *Apache) gatherUrl(addr *url.URL, acc telegraf.Accumulator) error {
+	var err error
```

Comment:
this is not good Go style to declare every single variable at the top before you use them. Try to refactor this to predeclare less variables. I think that all you really need to declare is `var client *http.Client`